benchmark/buffers/buffer-swap.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  method: ['swap16', 'swap32'],
+  len: [4, 64, 1024, 2056, 4096],
+  n: [1e7]
+});
+
+function main(conf) {
+  const method = Buffer.prototype[conf.method];
+  const len = conf.len | 0;
+  const n = conf.n | 0;
+  const buf = Buffer(len);
+  for (var i = 1; i <= len; i++)
+    buf[i - 1] = i;
+  bench.start();
+  for (i = 0; i < n; i += 1)
+    method.apply(buf);
```

Comment:
Maybe avoid using `apply` inside the bench start/end?

In [buffer-iterate.js](https://github.com/nodejs/node/blob/master/benchmark/buffers/buffer-iterate.js#L25) it just calls off the instance...
```javascript
const name = conf.method;
//...
for (i = 0; i < n; i += 1)
  method[name](buf);
```